node/middlewares/notify.ts

```diff
@@ -0,0 +1,26 @@
+import { json } from 'co-body'
+
+import type { OrderFormEvent } from '../typings/orderFormEvent'
+
+export async function notify(ctx: Context, next: () => Promise<unknown>) {
+  const {
+    clients: { events },
+    headers,
+  } = ctx
+
+  const orderFormEvents = (await json(ctx.req)).events as OrderFormEvent[]
+  const eventsPromise = orderFormEvents
+    .filter((e) => e.triggered)
+    .map((e) => {
+      events.sendEvent('', e.eventName, {
+        ...(e.eventObject as Object),
```

Comment:
You can remove this cast to `Object` if you change `eventObject`'s type from `unknown` to `Record<string, unknown>`.

```suggestion
        ...e.eventObject,
```